src/internal/components/login/SecurityCookieStoreLoginDialog.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { I18nextManager } from 'tdp_core';
+import { useOrdinoLogo } from '../../../hooks/useOrdinoLogo';
+
+function SecurityCookieStoreLoginDialog() {
+  const { status, value } = useOrdinoLogo();
+  return (
+    <div style={{ minHeight: 200, display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center' }}>
+      {status === 'success' && <img src={value.icon} style={{ width: '60px', marginBottom: '2em' }} alt={value.text} />}
```

Comment:
```suggestion
```

As discussed in the chat, I would remove the static logo as it might confuse users and does not add any value. Please remove the `useOrdinoLogo` from this file, too. You can keep the hook though.